.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - run: |
-          curl -sOLJ 'https://github.com/jqlang/jq/releases/download/jq-1.7.1/jq-linux-amd64'
+          curl -sOLJ 'https://github.com/jqlang/jq/releases/download/jq-1.8.0/jq-linux-amd64'
```

Comment:
@thaliaarchi a bit confused how your PR passed without this? 🤔 1.7.1 should have failed on the escaped comments lines or?